### DISCUSSION TITLE
表格自定义列能否加一个Required 特性？

### DISCUSSION THREAD

**Discussion by yangf85** [dotnetcore/BootstrapBlazor#3167] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

![image](https://github.com/dotnetcore/BootstrapBlazor/assets/42733580/413d2bb4-768f-4cff-a3cd-ee5d3299684e)
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/42733580/dae0b8f8-4240-4e03-8337-09cc9e62f27f)

### Describe the solution you'd like

有点不一致

**Comment by bb-auto** [comment]
@yangf85 Thank you for reporting. We will give triage later.

**Comment by ArgoZhang** [comment, marked as answer]
@yangf85 这个是不需要设置的啊。参考一下组件库内可验证组件，你有 Required 标签就可以了
